documentation/samplefiles/Microsoft.YourServiceName/stable/YYYY-MM-DD/scenarios/quickStart.yaml

```diff
@@ -1,34 +0,0 @@
-# yaml-language-server: $schema=https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/documentation/api-scenario/references/v1.2/schema.json
```

Comment:
This doc is used as the quick start for API scenarios. If you keep the API scenario docs, this one should be reverted too.